bittensor_cli/cli.py

```diff
@@ -2089,6 +2097,59 @@ def wallet_set_id(
             ask_for=[WO.HOTKEY, WO.NAME],
             validate=WV.WALLET_AND_HOTKEY,
         )
+
+        if not any(
+            [
+                display_name,
+                legal_name,
+                web_url,
+                riot_handle,
+                email,
+                pgp_fingerprint,
+                image_url,
+                info_,
+                twitter_url,
+            ]
+        ):
+            console.print("[yellow]All fields are optional. Press Enter to skip a field.[/yellow]")
+
+        display_name = display_name or typer.prompt(
+            "Display name", default="", show_default=False
+        )
+        legal_name = legal_name or typer.prompt(
+            "Legal name", default="", show_default=False
+        )
+        web_url = web_url or typer.prompt(
+            "Web URL", default="", show_default=False
+        )
+        riot_handle = riot_handle or typer.prompt(
+            "Riot handle", default="", show_default=False
+        )
+        email = email or typer.prompt(
+            "Email address", default="", show_default=False
+        )
+        pgp_fingerprint = pgp_fingerprint or typer.prompt(
+            "PGP fingerprint (Eg: A1B2 C3D4 E5F6 7890 1234 5678 9ABC DEF0 1234 5678)",
+            default="",
+            show_default=False,
+        )
+        image_url = image_url or typer.prompt(
+            "Image URL", default="", show_default=False
+        )
+        info_ = info_ or typer.prompt("Enter info", default="", show_default=False)
+        twitter_url = twitter_url or typer.prompt(
+            "Twitter (𝕏) URL", default="", show_default=False
+        )
+
```

Comment:
We should actually be size-checking here as well, but let's leave that as a TODO feature for next time.

Reply:
I did this in https://github.com/opentensor/btcli/pull/151